x/stakeibc/keeper/icacallbacks_claim_test.go

```diff
@@ -71,6 +125,30 @@ func (s *KeeperTestSuite) TestClaimCallback_Successful() {
 
 	_, found := s.App.RecordsKeeper.GetUserRedemptionRecord(s.Ctx(), initialState.callbackArgs.UserRedemptionRecordId)
 	s.Require().False(found, "record has been deleted")
+
+	// fetch the epoch unbonding record
+	epochUnbondingRecord1, found := s.App.RecordsKeeper.GetEpochUnbondingRecord(s.Ctx(), tc.initialState.epochNumber)
+	s.Require().True(found, "epoch unbonding record found")
+	epochUnbondingRecord2, found := s.App.RecordsKeeper.GetEpochUnbondingRecord(s.Ctx(), tc.initialState.epochNumber+1)
+	s.Require().True(found, "epoch unbonding record found")
+
+	// fetch the hzus
+	hzu1 := epochUnbondingRecord1.HostZoneUnbondings[0]
+	hzu2 := epochUnbondingRecord1.HostZoneUnbondings[1]
+	hzu3 := epochUnbondingRecord2.HostZoneUnbondings[0]
+	hzu4 := epochUnbondingRecord2.HostZoneUnbondings[1]
+
+	// check that hzu1 has a decremented amount
+	s.Require().Equal(hzu1.NativeTokenAmount-tc.initialState.decrementAmount, hzu1.NativeTokenAmount, "hzu1 amount decremented")
+	s.Require().Equal(hzu1.Status, recordtypes.HostZoneUnbonding_TRANSFERRED, "hzu1 status set to transferred")
+	// verify the other hzus are unchanged
+	s.Require().Equal(hzu2.NativeTokenAmount, hzu2.NativeTokenAmount, "hzu2 amount unchanged")
+	s.Require().Equal(hzu2.Status, recordtypes.HostZoneUnbonding_UNBONDED, "hzu2 status set to transferred")
+	s.Require().Equal(hzu3.NativeTokenAmount, hzu3.NativeTokenAmount, "hzu3 amount unchanged")
+	s.Require().Equal(hzu3.Status, recordtypes.HostZoneUnbonding_UNBONDED, "hzu3 status set to transferred")
+	s.Require().Equal(hzu4.NativeTokenAmount, hzu4.NativeTokenAmount, "hzu4 amount unchanged")
+	s.Require().Equal(hzu4.Status, recordtypes.HostZoneUnbonding_UNBONDED, "hzu4 status set to transferred")
```

Comment:
They check that the callback data isn't modifying any other records- I just added them so that if any code in callbacks _does_ start modifying other records, the unittest will fail

Reply:
I think it's better to keep the tests lean now and extend them if those fields ever became relevant in the calling function. Imo, this isn't that different than checking a deposit record field here or any other element from the state that isn't touched by this function atm.

Not to make too big a deal about this very nit thing haha, but I do think keeping the tests focused to the function in question is an important principle with these unit tests. Wdyt?